src/settings/Settings.tsx

```diff
@@ -810,6 +811,12 @@ export const SETTINGS: Settings = {
         default: true,
         invertedSettingName: "hideDisplaynameChanges",
     },
+    "showServerAclChanges": {
+        supportedLevels: LEVELS_ROOM_SETTINGS_WITH_ROOM,
+        displayName: _td("settings|show_server_acl_changes"),
+        default: true,
+        invertedSettingName: "hideServerAclChanges",
```

Comment:
this is redundant as there was no `hideServerAclChanges` that came before it that this setting needs to read in inverse for migrations